### DISCUSSION TITLE
add support for https://paylike.io/

### DISCUSSION THREAD

**Discussion by sysadminpower2019** [pretix/pretix#3800] [General]
Hello,

Since the paypal option is not viable for what is basically a small scale once a year event and we really would like to accept cards this provider seems to be the only way forward.

The documentation for pretix seems to imply that it is possible to accept various payment providers but the actual documentation does not show to implement for example giropay or visa payments.

It would seem a site like paylike.io is ideal for small scale events. It would be great if we could integrate this with pretix for these use cases.

**Comment by raphaelm** [comment]
Someone could surely create a plugin for paylike although it's not entirely clear to me why it is better suited to a small event than Stripe or PayPal

**Comment by sysadminpower2019** [comment]
@raphaelm siging up for paylike takes less then 3 minutes and just works.
Paypal and stripe require extensive setups, fees, etc.
Paylike promises fixed transaction costs which are perfect for a quick small scale event. 
also of note is that alf.io comes with some kind of revolut integration, and is a similar event management platform.

I was unable to get alfio to work but it might be working also investigating this revolution integration option

**Reply by pc-coholic** [reply]
Not to burst your bubble, but please have a very good look at https://paylike.io/account/approval too.

No payment provider - and I really mean **not a single one** - is above KYC regulation and card scheme regulations.

From what is written on paylike's page, you a) still have to be a business to use their services and b) need to provide appropriate documentation about your business and business case.

You might be able to start accepting money right away/within 3 minutes, but if you fail KYC, you will get your account revoked and the money is going back to your customers. That's by the way exactly the same as Stripe handles their onboarding, too.

Since I haven't worked personally with Paylike yet, I can't speak to how much PITA Paylike's KYC-process is. But to circle back to my statement at the top: there are only some may rules and regulations that every PSP has to follow, if they want to keep their card network accreditation.

Long story short: Integrating with another payment provider such as paylike is certainly a possibility and we're always happy to see more additions to the probably 100ish already existing pretix payment provider integrations. But onboarding with them will most certainly not make the legal hassle go away.

In fact, I am willing to bet, that clicking the "upgrade to business account"-button for an existing PayPal-account is easier than signing up for a new account. As long as we are staying within the "just signing up and ignoring the looming KYC-process" framework ;)
